src/Umbraco.Cms.Persistence.EFCore.SqlServer/Umbraco.Cms.Persistence.EFCore.SqlServer.csproj

```diff
@@ -3,6 +3,16 @@
     <Title>Umbraco CMS - Persistence - Entity Framework Core - SQL Server migrations</Title>
     <Description>Adds support for Entity Framework Core SQL Server migrations to Umbraco CMS.</Description>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Ensure NuGet vulnerability warnings are not errors in non-Release mode -->
+    <WarningsNotAsErrors>NU1901,NU1902,NU1903,NU1904</WarningsNotAsErrors>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+    <!-- Ensure NuGet vulnerability warnings are errors for Release mode -->
+    <WarningsAsErrors>
+      NU1901,NU1902,NU1903,NU1904
+    </WarningsAsErrors>
+  </PropertyGroup>
```

Comment:
Wouldn't it be easier to invert the check (and omit the platform) like this:
```suggestion
  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
    <!-- Ensure NuGet vulnerability warnings are not errors in Debug mode -->
    <WarningsNotAsErrors>NU1901,NU1902,NU1903,NU1904</WarningsNotAsErrors>
  </PropertyGroup>
```

Reply:
So, I created this pattern from the project file scenarios with existing warnings when we wanted to add more exceptions to the warning codes (as those warnings would need to be warnings as errors on all platforms), so I thought I'd keep these in their own separate property group if that makes sense? 

When there are no other exceptions though it's probably more elegant but I was wondering about keeping it the same across all.

Happy to try either way

Reply:
I've just tried (and applied 😇) it, although some additional changes were required to merge all `WarningsAsErrors`...

The NuGet vulnerability warnings are now suppressed on debug builds in the `Directory.Build.props` file, so they apply to all projects. However, any project that already set `WarningsAsErrors` was overwriting these, so I've appended them by using `<WarningsNotAsErrors>$(WarningsNotAsErrors),PROJECTERRORCODES</WarningsNotAsErrors>` 👍🏻

I've also formatted the comments and fixed some minor issues, as I also noticed 2 projects completely opted-out using `<TreatWarningsAsErrors>false</TreatWarningsAsErrors>`...

Reply:
Awesome, that formatting looks way more manageable now!

The ones with all being excluded was because there were so many different warning codes I couldn't find time to list them all separately, but if it still builds like that then that's cool!

Thanks for all the tweaks! Makes sense.